### PR TITLE
fix: Update Loki v3 Helm statefulset-ingester.yaml template

### PR DESCRIPTION
**Pull request by Arno-Pretorius**
Updating the "-ingester.ring.instance-availability-zone=zone-default" parameter.

**What this PR does / why we need it**:
This parameter is giving an error "loki ingester not starting: flag not defined: -ingester.ring.instance-availability-zone" on the ingester service as seen in this logged Github/Loki issue https://github.com/grafana/loki/issues/13061.

I Updated  the parameter to "-ingester.availability-zone" in my Live manifest and then my Loki Ingester deployed and started to make index entries etc.
This parameter is required I am not sure if the fix should rather be made on the ingester service itself and not in the statefulset to reflect the correct value.

**Which issue(s) this PR fixes**:
Fixes #<https://github.com/grafana/loki/issues/13061>

**Special notes for your reviewer**:

**Checklist**
- [X ] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [ ] Documentation added
- [ ] Tests updated
- [ ] Title matches the required conventional commits format, see [here](https://www.conventionalcommits.org/en/v1.0.0/)
  - **Note** that Promtail is considered to be feature complete, and future development for logs collection will be in [Grafana Alloy](https://github.com/grafana/alloy). As such, `feat` PRs are unlikely to be accepted unless a case can be made for the feature actually being a bug fix to existing behavior.
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/setup/upgrade/_index.md`
- [ X] For Helm chart changes bump the Helm chart version in `production/helm/loki/Chart.yaml` and update `production/helm/loki/CHANGELOG.md` and `production/helm/loki/README.md`. [Example PR](https://github.com/grafana/loki/commit/d10549e3ece02120974929894ee333d07755d213)
- [ ] If the change is deprecating or removing a configuration option, update the `deprecated-config.yaml` and `deleted-config.yaml` files respectively in the `tools/deprecated-config-checker` directory. [Example PR](https://github.com/grafana/loki/pull/10840/commits/0d4416a4b03739583349934b96f272fb4f685d15)

### CODE REVIEW

File: production/helm/loki/templates/ingester/statefulset-ingester.yaml [hunk 1]
```diff
@@ -87,7 +87,6 @@ spec:
           {{- end }}
           args:
             - -config.file=/etc/loki/config/config.yaml
```
Comment (lindeskar) [inline production/helm/loki/templates/ingester/statefulset-ingester.yaml:89]:
```suggestion
            - -config.file=/etc/loki/config/config.yaml
            - -ingester.availability-zone=zone-default
```

Comment (Arno-Pretorius) [reply production/helm/loki/templates/ingester/statefulset-ingester.yaml:89]: I tested this parameter in my live manifest and it works, if it is set to "-ingester.availability-zone=zone-default", my ingester pods are running fine now.

File: production/helm/loki/templates/ingester/statefulset-ingester.yaml
Comment (bulmust) [inline production/helm/loki/templates/ingester/statefulset-ingester.yaml:1]: Does

**Comment by bulmust** [comment]
LGTM. 

Does not need any if statement.

**Review by bulmust — APPROVED** [review]

**Review by lindeskar — APPROVED** [review]
Not sure how I became a reviewer of this. I don't think I have the power to let it be merged. 😄

**Comment by Ronan-WeScale** [comment]
@Arno-Pretorius accepte the Contributor license aggrement please

**Comment by Arno-Pretorius** [comment]
> @Arno-Pretorius accept the Contributor license agreement please

@Ronan-WeScale Done, thank you

**Comment by alexinthesky** [comment]
please merge this

**Comment by Arno-Pretorius** [comment]
@grafana/loki-team Can we get some action on this item please?

**Review by MasslessParticle — APPROVED** [review]
Thanks for the contribution!

**Comment by diranged** [comment]
@MasslessParticle  When can we get a patch release pushed with this fix? Without it, our team is completely blocked from the migration off of `loki-distributed`...

**Comment by Arno-Pretorius** [comment]
The fix has been published with in the latest release https://github.com/grafana/helm-charts/releases/tag/helm-loki-6.7.1

**Pull request merged**